wiki/Linux-Troubleshooting.md

```diff
@@ -21,6 +21,17 @@ If you have Amdvlk installed on your system, it overrides other vulkan drivers a
 
 Check if Amdvlk is installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall Amdvlk from your system.
 
+## Failed to create VAAPI encoder
```

Comment:
Put `(Fedora)` here so it will be immediately obvious first target users for this particular issue.

Reply:
Apologies for the delay. I believe I've done it now if you could check the latest commit.